### PR TITLE
WinJS version info telemetry changes

### PR DESCRIPTION
**Pull request by MythiliM**
Signed-off-by: Mythili Muruganathan <mythilim@microsoft.com>

### CODE REVIEW

File: src/js/WinJS/Utilities/_TelemetryImpl.js [hunk 1]
```diff
@@ -10,7 +11,7 @@ define([
 
     /// Make sure to use these hard-coded strings
     var MicrosoftGroup = "4f50731a-89cf-4782-b3e0-dce8c90476ba";
-    var MicrosoftKeyword = "0x2000000000000";
+    var MicrosoftKeyword = 0x200000000000;
```
Comment (rigdern) [inline src/js/WinJS/Utilities/_TelemetryImpl.js]: It looks like this number lost a 0. Was that intentional? How many zeroes should it have?

Comment (MythiliM) [inline src/js/WinJS/Utilities/_TelemetryImpl.js]: Yes, it is intentional.  The number has apparently changed (they will still support the old number for some time but it is recommended that we change to this number).  Also, it should be an integer & not a string for telemetry events to be uploaded.  Thanks.

**Comment by rigdern** [comment]
Looks good to me :shipit:

**Comment by xirzec** [comment]
The changes look good to me. Did we confirm that the WinRT API actually gets called correctly?

**Comment by MythiliM** [comment]
Yes of course, I confirmed that the WinRT API sends telemetry out.  Thanks.

**Pull request merged**